pkg/model/model.go

```diff
@@ -76,6 +81,32 @@ func (m *Model) crdNames() []string {
 	return crdConfigs
 }
 
+func CheckIfFieldExists(
+	field map[string]*Field,
+	currValue string,
+) bool {
+	_, ok := field[currValue]
+	if ok {
+		return true
+	}
+	return false
+}
+
+func CheckType(
```

Comment:
Same comment here, i don't think we need this function.. or the least it should be named `isShapeStructure` .. and still not very convinced of moving this logic to a function

Reply:
Have removed the function and added the logic to the calling function